dbms/src/Storages/DeltaMerge/File/DMFile.cpp

```diff
@@ -712,33 +712,24 @@ std::set<UInt64> DMFile::listAllInPath(
         // or you may delete some writing files
         if (options.clean_up)
         {
-            if (startsWith(name, details::FOLDER_PREFIX_WRITABLE))
+            if (startsWith(name, details::FOLDER_PREFIX_WRITABLE) || startsWith(name, details::FOLDER_PREFIX_DROPPED))
             {
-                // Clear temporary files
-                const auto full_path = parent_path + "/" + name;
-                if (Poco::File temp_file(full_path); temp_file.exists())
-                    temp_file.remove(true);
-                LOG_FMT_WARNING(log, "Existing temporary dmfile, removed: {}", full_path);
-                continue;
-            }
-            else if (startsWith(name, details::FOLDER_PREFIX_DROPPED))
-            {
-                // Clear deleted (maybe broken) DTFiles
+                // Clear temporary/deleted (maybe broken) files
+                // The encryption info use readable path. We are not sure the encryption info is deleted or not.
+                // Try to delete and ignore if it is already deleted.
                 auto res = try_parse_file_id(name);
                 if (!res)
                 {
-                    LOG_FMT_INFO(log, "Unrecognized dropped DM file, ignored: {}", name);
+                    LOG_FMT_INFO(log, "Unrecognized temporary or dropped dmfile, ignored: {}", name);
```

Comment:
Use `LOG_INFO` instead. We will remove `LOG_FMT_XXX` which is the same as `LOG_XXX`.

Reply:
Fixed.